app/modules/emails/utils.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+import logging
+import re
+
+log = logging.getLogger(__name__)
+
+EMAIL_PATH = 'app/modules/emails/templates/en/'
+DIGEST_EMAIL_PATH = f'{EMAIL_PATH}digest/'
+
+
+class EmailUtils(object):
+    @classmethod
+    def build_email(cls, template, replacements, is_digest=False):
+        path = DIGEST_EMAIL_PATH if is_digest else EMAIL_PATH
+        template_filename = f'{path}{template}.html'
+        with open(template_filename) as file:
+            text = file.read()
+            for key in replacements.keys():
+                text = re.sub(f'@{key}@', replacements[key], text)
+        return text
+
+    @classmethod
+    def _send_email_with_mailchimp(cls, email_contents):
+        # TODO In future this will link in to the library to do this
+        pass
+
+    @classmethod
+    def send_email(cls, email_contents):
```

Comment:
i wonder if this needs added: `recipient`, `subject`, maybe `sender` ?

`subject` is a weird one.  should that be its own template?  i think we hard-coded in old world.  :cry:

Reply:
Very good points. 
I think NOTIFICATION_FIELDS needs to be replaced with NOTIFICATION_CONFIG indexed on NotificationType containing 
mandatory_fields
email template filename
email subject.

Should sender_email be a mandatory field for all Notifications? I think it should now.

Reply:
i agree mandatory, even if that means we stuff in some kinda `do-not-reply@wildme.org` or something.  most outgoing mail services wants _a_ sender.